video_xblock/static/js/player_state.js

```diff
@@ -68,8 +92,8 @@ var saveState = function(){
     'transcriptsEnabled': player.transcriptsEnabled,
     'captionsEnabled': player.captionsEnabled,
     'captionsLanguage': player.captionsLanguage,
+    'downloadTranscriptUrl': downloadTranscriptUrl
```

Comment:
You know. After a second thought, I think it's better to not add tran url into a state. Because it's not a part of player's state.
Let's better pass it as new attribute to an object passed into `postMessage()`